scout/server/blueprints/cases/templates/cases/individuals_table.html

```diff
@@ -126,7 +126,7 @@
                   </div>
                 </label>
                 {% if gens_info.display %}
-                  <a style="font-size: .8rem;" class="btn btn-secondary btn-sm" target="_blank" href="http://{{gens_info.host}}/{{ind.display_name}}?&genome_build={{case.genome_build}}">CN profile</a>
+                  <a style="font-size: .8rem;" class="btn btn-secondary btn-sm" target="_blank" href="http://{{gens_info.host}}/{{ind.display_name}}?&genome_build={{case.genome_build}}&case_id={{case.case_id}}">CN profile</a>
```

Comment:
Code-wise it looks good. Have you tried it on stage?

Reply:
Not yet! Our local fork still uses `case_name`, but @raysloks relevantly suggested switching to `case_id` . Are you on that one, @raysloks, or should we PR it? 😊

Reply:
Yeah, I'm going to be pushing the swap to `case_id` on Gens quite soon.